src/App.tsx

```diff
@@ -2,108 +1,64 @@
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalItems = items.length;
+
+  const from = (currentPage - 1) * Number(perPage) + 1;
+  const to =
+    currentPage * Number(perPage) > 42 ? 42 : currentPage * Number(perPage);
```

Comment:
The magic number `42` here is a potential source of bugs if the total number of items changes. It would be better to use `totalItems` instead to make the code more maintainable.